be/src/common/daemon.cpp

```diff
@@ -203,29 +205,42 @@ void Daemon::memory_maintenance_thread() {
         doris::MemInfo::refresh_allocator_mem();
 #endif
         doris::MemInfo::refresh_proc_mem_no_allocator_cache();
-        LOG_EVERY_N(INFO, 10) << MemTrackerLimiter::process_mem_log_str();
 
         // Refresh mem tracker each type metrics.
         doris::MemTrackerLimiter::refresh_global_counter();
-        if (doris::config::memory_debug) {
-            doris::MemTrackerLimiter::print_log_process_usage("memory_debug", false);
-        }
         doris::MemTrackerLimiter::enable_print_log_process_usage();
 
         // If system available memory is not enough, or the process memory exceeds the limit, reduce refresh interval.
         if (doris::MemInfo::sys_mem_available() <
                     doris::MemInfo::sys_mem_available_low_water_mark() ||
             doris::MemInfo::proc_mem_no_allocator_cache() >= doris::MemInfo::mem_limit()) {
-            interval_milliseconds = 100;
+            doris::MemTrackerLimiter::print_log_process_usage("process full gc", false);
```

Comment:
Looks like it will print too many logs when memory is insuffient?
Because the interval is only 500ms?

Reply:
100ms print once

Reply:
done, optimized log printing interval